### PR TITLE
Added a config to replace existing worlds' vanilla nether biome providers with ours.

### PR DESCRIPTION
**Pull request by SwanX1**
NOTE: Config doesn't show up in the GUI because it's a dangerous feature and can break worlds 
Thanks to @Vazkii

### CODE REVIEW

File: src/main/java/org/infernalstudios/infernalexp/InfernalExpansion.java [hunk 1]
```diff
@@ -73,6 +74,7 @@
 import org.infernalstudios.infernalexp.network.IENetworkHandler;
 import org.infernalstudios.infernalexp.util.CompatibilityQuark;
 import org.infernalstudios.infernalexp.util.ModSpawnEggItem;
+import org.infernalstudios.infernalexp.util.NoiseChunkGeneratorUtil;
```
Comment (Hellion-Games) [inline src/main/java/org/infernalstudios/infernalexp/InfernalExpansion.java]: Why did you add imports but not change anything else in this file?

File: src/main/java/org/infernalstudios/infernalexp/config/CommonConfig.java [hunk 2]
```diff
@@ -326,6 +327,15 @@
             .translation(InfernalExpansion.MOD_ID + ".config.tooltip.biomesList")
             .define("biomesList", "");
 
+        builder.push("Dangerous Configs");
+
+        replaceNetherBiomeProvider = builder
+            .comment("Replaces existing worlds' vanilla nether biome provider with ours.\nTHIS CAN BREAK YOUR WORLD, DO NOT TURN ON IF YOU DON'T KNOW WHAT YOU'RE DOING")
+            .translation(InfernalExpansion.MOD_ID + ".config.replaceNetherBiomeProvider")
```
Comment (Hellion-Games) [inline src/main/java/org/infernalstudios/infernalexp/config/CommonConfig.java:334]: Does it need a translation since it's not in the config GUI?

File: src/main/java/org/infernalstudios/infernalexp/config/CommonConfig.java [hunk 3]
```diff
@@ -378,5 +388,7 @@
             .comment("Determines if bricks are throwable or not")
             .translation(InfernalExpansion.MOD_ID + ".config.tooltip.useThrowableBricks")
             .define("useThrowableBricks", true);
+
+        builder.pop();
```
Comment (SwanX1) [inline src/main/java/org/infernalstudios/infernalexp/config/CommonConfig.java:392]: note: this was added because it was missing for the miscellaneous configs

**Review by Dariensg — CHANGES_REQUESTED** [review]

File: src/main/java/org/infernalstudios/infernalexp/InfernalExpansion.java (hunk 1 above)
Comment (SwanX1) [reply src/main/java/org/infernalstudios/infernalexp/InfernalExpansion.java]: ah, this is an artifact of testing, will remove

File: src/main/java/org/infernalstudios/infernalexp/config/CommonConfig.java (hunk 2 above)
Comment (SwanX1) [reply src/main/java/org/infernalstudios/infernalexp/config/CommonConfig.java:334]: it's best for parity with other configs in code, but the lang files omit it because it doesn't show up in the gui

**Review by Dariensg — APPROVED** [review]
LGTM
